components/org.wso2.carbon.deployment.engine/src/main/java/org/wso2/carbon/deployment/engine/Constants.java

```diff
@@ -26,7 +26,7 @@
  */
 public final class Constants {
 
-    public static final String DEPLOYMENT_YAML_CONFIGURATION_FILE = "deployment.yml";
+    public static final String DEPLOYMENT_YAML_CONFIGURATION_FILE = "deployment.yaml";
```

Comment:
This isn't the configuration file we will use anymore. Please update the file name as appropriate.